src/Console/Arguments.php

```diff
@@ -113,12 +147,13 @@ public function hasArgument(string $name): bool
     }
 
     /**
-     * Returns positional argument value by name or null if doesn't exist
+     * Internal method to get argument from name.
+     * Avoid duplication code for getArgument() and getArrayArgument().
      *
-     * @param string $name The argument name to check.
-     * @return string|null
+     * @param string $name Argument name.
+     * @return array|string|null
      */
-    public function getArgument(string $name): ?string
+    private function _getArgument(string $name): array|string|null
```

Comment:
We still have here `array|scalar` added, which in the other methods the point was to avoid this.
Is there a way we could do this here too and require the specific methods to be used for array, and getting raw string returned for the others if used?

Reply:
this method was removed and code rewrited and only 5 duplicated lines.